test/functional/forall/test-forall-atomic-utils.hpp

```diff
@@ -0,0 +1,88 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#ifndef __TEST_FORALL_ATOMIC_UTILS_HPP__
+#define __TEST_FORALL_ATOMIC_UTILS_HPP__
+
+#include "RAJA/RAJA.hpp"
+
+#include "test-forall-utils.hpp"
+
+using SequentialForallAtomicExecPols =
+  camp::list<
+              RAJA::seq_exec,
+              RAJA::loop_exec
+              //RAJA::simd_exec not expected to work with atomics
+            >;
+
+using SequentialAtomicPols =
+  camp::list<
+#if defined(RAJA_TEST_EXHAUSTIVE)
+              RAJA::auto_atomic,
+              RAJA::builtin_atomic,
+#endif
+              RAJA::seq_atomic
+            >;
+
+#if defined(RAJA_ENABLE_OPENMP)
+using OpenMPForallAtomicExecPols =
+  camp::list<
+#if defined(RAJA_TEST_EXHAUSTIVE)
+              RAJA::omp_for_nowait_exec,
+              RAJA::omp_parallel_for_exec,
+#endif
+              RAJA::omp_for_exec
+              //RAJA::omp_parallel_exec<RAJA::seq_exec> not expected to work with atomics
```

Comment:
It would be more clear to say that this is expected to work with atomics, but that it will not pass the test because the policy does not run the same work as the other policies.

Reply:
Thanks, changed this explanation a bit.